.github/workflows/docker.yml

```diff
@@ -0,0 +1,90 @@
+# SPDX-FileCopyrightText: 2021-2024 Nextcloud GmbH and Nextcloud contributors
+# SPDX-License-Identifier: MIT
+
+name: Docker image
+
+on:
+  release:
+    types: [published]
+  push:
+    branches:
+        - main
+        - master
+        - stable*
+
+env:
+  GHCR_REPO: ghcr.io/${{ github.repository_owner }}/whiteboard
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+    timeout-minutes: 30
+
+    strategy:
+      matrix:
+        platform:
+          - linux/amd64
+          - linux/arm64
+
+    steps:
+      - name: Set vars
+        id: vars
+        run: |
+          echo "version_channel=${{ github.event_name == 'release' && 'release' || 'daily' }}" >> $GITHUB_OUTPUT
+          echo "version=${{ github.sha }}" >> $GITHUB_OUTPUT
```

Comment:
so not version tag like e.g. v1.0.0 or comparable?

Reply:
Ah right we should also tag those 👍